src/main/cluster.ts

```diff
@@ -80,20 +84,15 @@ export class Cluster implements ClusterModel {
 
   @action
   async init(port: number) {
-    if (this.initialized) {
```

Comment:
Is it now save to init twice?

Reply:
No, init used only in single place in cluster-manager with that check